docs/INTEGRATION.md

```diff
@@ -261,6 +261,14 @@ if docker run --rm -i hadolint/hadolint < "%d/%f"
 | grep -EC100 ':WARNING:' ; then exit 1 ; else exit 0 ; fi
 ```
 
+## Other
+
+Following section contains different angles on `hadolint` usage 
+that go outside of [code review](#code-review) or [editors' integration](#editors). 
+
+- [hadolint-shim](https://github.com/kornicameister/hadolint-shim) - use `hadolint` 
+  as normal executable on systems where static binaries are not accessible
```

Comment:
Maybe it should be added that it uses docker and why you want to use that but I am not to sure.

Reply:
It might be, not saying that it shouldn't. But if we do that here, shouldn't similar explanation go to each entry.  For instance, you have `ale` for `vim` that I am using as well but there's no such explanation there + and it _can_ use `docker` variant too. I mean, mine is using `docker`, other integration might use that, but how `hadolint` is used in specific integration goes a little bit outside of scope of `hadolint` itself, doens't? What is important is that they make life easier.

Reply:
Though, maybe `accessible` ought to be changed to `available` to emphasize that binaries are available only for certain platforms as mentioned [here](https://github.com/hadolint/hadolint#install).

Reply:
I agree @kornicameister